### ISSUE TITLE
Associate error messages

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1670]
Relevant Components:

- All error messages

Description
Error messages are easy enough to place when the user can see. However, when they cannot, error messages can be confusing unless they are programmatically associated with the element for which an alert is triggered. 

There are many ways to do this, listed [+on w3’s website+|https://www.w3.org/WAI/tutorials/forms/notifications/].
Current Code
```
<input>


<div class="alert ...”>
...
</div>
```

Solution
Add a way for a screen reader (read: program) to understand the relationship between the error message and the input. The simplest way, though not the most effective, is to list the id of the error message in an {{aria-describedby}} attribute on the affected input. Read [w3’s guide to user alerts|https://www.w3.org/WAI/tutorials/forms/notifications/] to learn more.

Make sure the error message also has {{role=”alert”}}. This allows the screen reader to announce it as an alert, decreasing the cognitive load on the user, and making a more accessible experience.

Recommended Code
```
<input aria-describedby="error-message">


<div id="error-message" role="alert" class="alert ...">
...
</div>
```

**Comment by jcreel** [comment]
The context for this is the field-specific form validation errors, e.g. during student submission workflow.  Size this supposing we depend on #1668 which would provide ids on the fields.
